sofiadinizms/vida-de-inseto
# Feature/level 4

# What I did

* Created the Level 4 mechanics: leave your pet alone by leaving the app and coming back a while later to see it welcome you back.

# How to test

* Git pull this branch;
* Run it on an Xcode simulator;
* Check if the text onscreen says "Tô de abuso, preciso de um minutinho.";
* If so, leave the app by hitting the home button, or changing apps, or sliding up and going to the home page. This will make the app go to "Background" mode, and you can check this by looking at the console log print that says "Background";
* Come back to the app about 30 seconds later. The text should now say "Já tá de volta? Me deixa em paz um minutinho";
* Leave it again and come back 30 to 40 seconds later. If the difference between the time you first left the app and your latest return is over 1 minute, the text should change to: "Agora sim, bebê! Bora dar uma subidinha!".

If you managed to successfully complete all these tasks as described above, then everything is working correctly. If not, you're a liar and your pants are truly on fire.
